### PR TITLE
Support tag ignore and extra triggers args

### PR DESCRIPTION
**Pull request by yaronha**

### CODE REVIEW

File: nuclio/utils.py [hunk 1]
```diff
@@ -117,6 +118,7 @@ def parse_export_line(args):
     parser.add_argument('--archive', '-a', action='store_true', default=False)
     parser.add_argument('--verbose', '-v', action='store_true', default=False)
     parser.add_argument('--kind', default=None)
+    parser.add_argument('--ignored-tags', default='')
```
Comment (liranbg) [inline nuclio/utils.py:121]: provide help for example?

File: nuclio/export.py [hunk 2]
```diff
@@ -164,7 +180,8 @@ def scan_notebook_cells(self, config, cells):
         for cell in filter(is_code_cell, cells):
             code_in_cell_with_annotation = ''
             code = cell['source']
-            if has_ignore(code):
+            tags = get_in(cell, 'metadata.tags')
+            if has_ignore(code) or ignore_tagged_cell(tags, ignored_tags):
```
Comment (liranbg) [inline nuclio/export.py:184]: mind you adding a test to make sure nothing breaks this feature?

**Review by liranbg — COMMENTED** [review]
looking legit, left some comments with improvements

File: nuclio/export.py [hunk 3]
```diff
@@ -49,6 +49,7 @@
 has_ignore = re.compile(rf'{annotation_prefix}ignore').search
 has_start = re.compile(rf'{annotation_prefix}start-code[ \t]*(?P<name>([\S]*))?').search
 has_end = re.compile(rf'{annotation_prefix}end-code[ \t]*(?P<name>([\S]*))?').search
+default_ignored_tag = 'mlrun-ignore'
```
[the remark this answers is not in the document]
Comment (yaronha) [reply nuclio/export.py:52]: i rather not make it X-.. support a list with both mlrun and nuclio as default (`default_ignored_tags = 'mlrun-ignore;nuclio-ignore'`)

File: nuclio/export.py [hunk 4]
```diff
@@ -74,6 +75,20 @@ def create_logger():
...
+    if ignored_tags:
+        ignored_tags = ignored_tags.split(";")
+    return ignored_tags + [default_ignored_tag]
+
+
+def ignore_tagged_cell(tags, tags_to_ignore):
+    for tag in tags or []:
+        if tag in tags_to_ignore:
+            return True
+    return False
```
[the remark this answers is not in the document]
Comment (yaronha) [reply nuclio/export.py:89]: done

File: nuclio/utils.py (hunk 1 above)
Comment (yaronha) [reply nuclio/utils.py:121]: this is for the megic command, there is no help for support for it AFAIK

**Review by liranbg — APPROVED** [review]

File: nuclio/triggers.py [hunk 5]
```diff
@@ -203,14 +230,15 @@ def __init__(
...
         if pollingIntervalMS:
             struct["attributes"]["pollingIntervalMs"] = pollingIntervalMS
-        access_key = access_key \
-            if access_key else environ.get("V3IO_ACCESS_KEY")
+        access_key = access_key if access_key else environ.get("V3IO_ACCESS_KEY")
         if not access_key:
-            raise ValueError('access_key must be set '
-                             '(via argument or environ V3IO_ACCESS_KEY)')
+            raise ValueError(
+                "access_key must be set " "(via argument or environ V3IO_ACCESS_KEY)"
```
Comment (AlonMaor14) [inline nuclio/triggers.py:236]:
```suggestion
                "access_key must be set (via argument or environ V3IO_ACCESS_KEY)"
```

Comment (yaronha) [reply nuclio/triggers.py]: fixed (black auto format error)

**Pull request merged**